js/recruitment.js

```diff
@@ -1,5 +1,136 @@
 import { contactInfoRequest } from "./contactInfo.js";
+import { recruitmentInfoRequest } from "./recruitmentInfo.js";
 
+/*
+ * Trying to augment this version to both support API fetch of
+ * contact info and recruitment text
+ */
+
+let contactInfo = null;
+let recruitmentInfo = null;
+
+const contactLoaded = (data) => {
+  contactInfo = data;
+  if(recruitmentInfo != null) {
+    createHTML(contactInfo, recruitmentInfo);
+  }
+}
+
+const recruitmentLoaded = (data) => {
+  recruitmentInfo = data;
+  if(contactInfo != null) {
+    createHTML(contactInfo, recruitmentInfo);
+  }
+}
+
+// Listen for both fetches
+document.addEventListener("DOMContentLoaded", event => {
+  contactInfoRequest.then(contactLoaded);
+  recruitmentInfoRequest.then(recruitmentLoaded);
+});
+
+
+// We create a reusable template for the content
+// This approach makes it easier to avoid .innerHMTL to the benefit of .textContent,
+// reducing the need for trusting/sanitising our input. It's also faster (presumably)!
+// Main drawback is somewhat tedious manual DOM tree building.
+function createWrapperTemplate() {
+  let baseWrapperNode = document.createElement('div');
+  baseWrapperNode.className = "band";
+
+  let wrapperHeadingTop = document.createElement('h3');
+  baseWrapperNode.appendChild(wrapperHeadingTop);
+
+  let wrapperHeadingSub = document.createElement('h4');
+  wrapperHeadingSub.textContent = 'Ledige plasser:';
+  baseWrapperNode.appendChild(wrapperHeadingSub);
+
+  let longTextElem = document.createElement('p');
+  longTextElem.className='longer-text';
```

Comment:
Jeg vet ikke om vi har noen style guide for Låfte kode, men du bruker space før og etter `=` alle andre steder, så for consistency bør du ha det her også.

Reply:
Enig